pkgs/applications/networking/browsers/misc/widevine-cdm.nix

```diff
@@ -1,34 +1,70 @@
 { lib
 , stdenv
+, fetchurl
 , fetchzip
+, fetchFromGitHub
+, squashfsTools
+, curl
+, python3
 }:
 
 stdenv.mkDerivation rec {
   pname = "widevine-cdm";
   version = "4.10.2710.0";
+  lacrosVersion = "120.0.6098.0";
 
-  src = fetchzip {
+  widevineInstaller = if stdenv.isAarch64 then fetchFromGitHub {
+      owner = "AsahiLinux";
+      repo = "widevine-installer";
+      rev = "7a3928fe1342fb07d96f61c2b094e3287588958b";
+      sha256 = "sha256-XI1y4pVNpXS+jqFs0KyVMrxcULOJ5rADsgvwfLF6e0Y=";
+    } else null;
+
+  src = if stdenv.isAarch64 then fetchurl {
+    url = "https://commondatastorage.googleapis.com/chromeos-localmirror/distfiles/chromeos-lacros-arm64-squash-zstd-${lacrosVersion}";
+    hash = "sha256-OKV8w5da9oZ1oSGbADVPCIkP9Y0MVLaQ3PXS3ZBLFXY=";
+  } else fetchzip {
     url = "https://dl.google.com/widevine-cdm/${version}-linux-x64.zip";
     hash = "sha256-lGTrSzUk5FluH1o4E/9atLIabEpco3C3gZw+y6H6LJo=";
     stripRoot = false;
   };
 
+  buildInputs = [ squashfsTools curl python3 ];
+
+  unpackPhase = if stdenv.isAarch64 then ''
+    curl -# -o lacros.squashfs "file://$src"
```

Comment:
Why did you copy this from upstream? Just copy the file...?

Reply:
Could you elaborate? unsquashfs wouldn't work without renaming using curl like widevine-installer does, but I only tried a few times, maybe it makes sense to figure this part out and remove curl from the buildInputs if that's what you mean.

Reply:
curl in this case is a glorified copy or symlink. It makes sense to use it upstream because they're downloading from a URL. Please try `cp` or `ln`ing the file to see if that works, it should.